src/test/java/gov/cabinetoffice/gap/adminbackend/services/SubmissionsServiceTest.java

```diff
@@ -524,30 +524,9 @@ void getCompletedSubmissionExports() {
         }
 
         @Test
-        void getCompletedSubmissionExportsWithDecodedLabels() {
```

Comment:
deleting this test as it's no longer needed - we're processing an s3 object key rather than a URL